### ISSUE TITLE
motion has 'momentum' in forward kinematics

### ISSUE THREAD

**Issue by koichishi** [dilevin/computer-graphics-kinematics#30]
when I release the robot hand it would 'shake' to left and right for a while and then goes to the desired pose.
I guess this is caused by the gradient descent since we stop searching for the learning rate once the corresponding f(z) is less than the current f(z).
I am not sure if this is fine, or should we find the step with absolute minimum f(z) within 0-10000?

Thanks!

**Comment by dilevin** [comment]
To be clear, line search terminates when the current f is less than the previous f. Gradient descent (the outer loop) does not use this as a termination criteria

**Comment by koichishi** [comment]
Thank you
